### DISCUSSION TITLE
Possibility to control MHP30 remotely?

### DISCUSSION THREAD

**Discussion by nerodesai** [Ralim/IronOS#1816] [General]
Is there a way I can control the temperature or functionality on the MHP30 device remotely without having to touch the device? The device gets really hot and we want to avoid any burns or injury and wanted to check if the controls could be done using a remote cli or some kind of web ui? 

Can you please let me know if this is possible to achieve or if there is an existing solution to this issue? @Ralim

**Comment by Ralim** [comment]
There is no existing solution for the MHP30. I do not have any means planned either at this point in time.

I would suggest using the temperature profiles to automate adjustments or just turning it on before placing the board on top.
